Tests/configure_and_test_integration_instances.py

```diff
@@ -709,12 +761,14 @@ def main():
         instance_name = instance.get('name', '')
         msg = 'Testing ("Test" button) for instance "{}" of integration "{}".'.format(instance_name,
                                                                                       integration_of_instance)
-        print(msg)
+        prints_manager.add_print_job(msg, print_color, 0, LOG_COLORS.GREEN)
+        prints_manager.execute_thread_prints(0)
         # If there is a failure, __test_integration_instance will print it
         success = __test_integration_instance(testing_client, instance, prints_manager)
         prints_manager.execute_thread_prints(0)
         if not success:
             preupdate_fails.add((instance_name, integration_of_instance))
+        preupdate_success.add((instance_name, integration_of_instance))
```

Comment:
It looks like should be in an else block. OW, all instances (failed and successful ones) will be added to `preupdate_success`